transit_odp/fares/views/update.py

```diff
@@ -99,18 +100,19 @@ def get(self, request, *args, **kwargs):
 
         # prevent user from trying to update dataset with existing draft, regardless of
         # how they got to this page
-        if self.object.dataset.revisions.filter(
-            status__in=[FeedStatus.draft.value, FeedStatus.success.value]
-        ).exists():
-            return redirect(
-                "fares:feed-draft-exists",
-                pk=self.object.dataset_id,
-                pk1=self.object.dataset.organisation_id,
-            )
-        else:
-            # Ensure GET returns the preview step
-            self.storage.reset()
-            return self.render(self.get_form())
+        # if self.object.dataset.revisions.filter(
+        #     status__in=[FeedStatus.draft.value, FeedStatus.success.value]
+        # ).exists():
+        #     return redirect(
+        #         "fares:feed-draft-exists",
+        #         pk=self.object.dataset_id,
+        #         pk1=self.object.dataset.organisation_id,
+        #     )
+        # else:
+        # Ensure GET returns the preview step
+        self.object.dataset.start_revision()
+        self.storage.reset()
+        return self.render(self.get_form())
```

Comment:
I think there is no harm in keeping the feed draft exists view if a user tries to create a new draft when there is already a draft existing.
Although, This will not be reproduced as the previous draft is been deleted.

Reply:
@surabhika Have tested and pushed the code back in.

Reply:
Not that it matters much, but the return should be inside else block